packages/components/dropdown/src/dropdown.vue

```diff
@@ -148,12 +147,9 @@ export default defineComponent({
     const scrollbar = ref(null)
     const currentTabId = ref<string | null>(null)
     const isUsingKeyboard = ref(false)
-    const triggerKeys = [
-      EVENT_CODE.enter,
-      EVENT_CODE.numpadEnter,
-      EVENT_CODE.space,
-      EVENT_CODE.down,
-    ]
+    const triggerKeys = computed(() =>
```

Comment:
The variable name may be changed to avoid duplication with prop.

```suggestion
    const _triggerKeys = computed(() =>
```

Reply:
I have already renamed triggerKeys to dropdownTriggerKeys.